packages/wrangler/package.json

```diff
@@ -36,7 +36,7 @@
     "cli"
   ],
   "dependencies": {
-    "esbuild": "0.14.14",
+    "esbuild": "0.14.23",
```

Comment:
Is there are reason to use exact version? Esbuild looks stable in patch versions. Vite also uses caret here.
https://unpkg.com/browse/vite@2.8.4/package.json

Reply:
esbuild pre v1 still ships breaking changes in minor releases, which is fine, but can mean that it could potentially break our users without us ever changing code. So I like being explicit about its updates. eg - https://github.com/cloudflare/wrangler2/pull/343 this update could have broken us.